### ISSUE TITLE
Server restart

### ISSUE THREAD

**Issue by OldManCraft** [LatvianModder/FTBUtilities#366]
Newest version,
forge 12.18.3.2239, 
Server when in restart works fine but just stops, does not restart.

**Comment by LatvianModder** [comment]
Newest, 3.4.0+? We have a server that restarts just fine, weird

**Comment by OldManCraft** [comment]
yes 3.4.0. I left for a few hours and came beck to a stopped server. No crash.

**Comment by LatvianModder** [comment]
Oh yeah, the 'restart' doesnt actually restart the server, you need a .sh or .bat script to restart it back up

**Comment by OldManCraft** [comment]
your mod has always restarted the server with no other script. you changed something from the last version that worked fine.

**Comment by OldManCraft** [comment]
Ill just go back to the last version that worked fine.

**Comment by LatvianModder** [comment]
it never did that.. Its always ServerStart.sh / .bat doing it. Java physically cant restart itself, without help of something else
